docs/source/examples/first_steps/beyond_pipeline.py

```diff
@@ -0,0 +1,64 @@
+"""Beyond the pipeline."""
+
+# do not complain about prints in this file
+# ruff: noqa: E402
+
+# %%
```

Comment:
these allow executing individual code parts jupyter cells. However, they also make including the files a bit more messy (you have to specify line ranges in rst, which may change when formatting changes, ...)

Reply:
this seems like a huge liability, especially if we ever want to change these. maybe there's some kind of locking/hashing mechanism we can check?